francis/split_filter.py

```diff
@@ -14,14 +15,16 @@ def call(df):
 
 def __split(pre_df):
     labeled = []
+    bar = Bar("splitting audio into 5 second chunks", max=len(pre_df))
     for i, row in pre_df.iterrows():
         split_buffer = __split_buffer(row["audio_buffer"], 22050, 5)
         filtered_buffer = __filter_chunks(
             split_buffer, type="quartile", cutoff="default"
         )
         for buffer in filtered_buffer:
             labeled.append((row["label"], buffer))
-
+        bar.next()
+    bar.finish()
     return pd.DataFrame(labeled, columns=["label", "audio_buffer"])
 
```

Comment:
Would there be a way of doing the bar class like the spinner class?
If it were me I would make a bar object and pass it in as an argument
```
bar = Bar("splitting audio", max=len(pre_df))
split_filter.call(pre_df, loading_object=bar)
```
this way if in future you wanted interchangeable loading objects you could **without changing any split_filter code**
you could also put in defaults to handle if the loading_object=None (aka didnt exist) meaning that you don't have to have a loading function at all.
This is the power of the single responsibility principle.

You don't have to change it, as this isnt going to be shipped to some google product
but its food for thought! 🙂